instrumentation/opentelemetry-instrumentation-tornado/src/opentelemetry/instrumentation/tornado/__init__.py

```diff
@@ -289,29 +319,28 @@ def _wrap(cls, method_name, wrapper):
     wrapt.apply_patch(cls, method_name, wrapper)
 
 
-def _prepare(tracer, request_hook, func, handler, args, kwargs):
-    start_time = time_ns()
+def _prepare(self, request_hook, func, handler, args, kwargs):
     request = handler.request
     if _excluded_urls.url_disabled(request.uri):
         return func(*args, **kwargs)
-    ctx = _start_span(tracer, handler, start_time)
+    ctx = _start_span(self, handler)
     if request_hook:
         request_hook(ctx.span, handler)
     return func(*args, **kwargs)
```

Comment:
Shouldn't we be tracking the request information here instead of `start_span` since this is where the request is actually being made?

Reply:
That makes sense, ill fix it